content/06.multiple-approaches-required.md

```diff
@@ -1,36 +1,29 @@
 
-<!-- TODO: This needs to explicitly hit regularization, other data, and prior knowledge points!--> 
+As discussed in the transfer learning section, feature-representation-transfer is an approach wherein a model learns a representation of some collection of data and those features are applied or transferred to a target domain.
+This can be a powerful tool for rare diseases if the model used requires more samples than available samples that assay the disease of interest, but we have a collection of data that is related.
+Whether our research goal is biological discovery or prediction, we want the representation that a model learns to be _useful_.
+In addition to reducing model complexity as discussed earlier, regularization can aid in useful representations by putting some constraints on what is learned by a model.
 
-<!-- From representation learning -->
-
-In the rare disease domain, Dincer et al. leveraged publicly available acute myeloid leukemia (AML) gene expression data to improve the prediction of _in vitro_ drug responses [@doi:10.1101/278739].
+In a method termed DeepProfile, Dincer et al. leveraged publicly available acute myeloid leukemia (AML) gene expression data to improve the prediction of _in vitro_ drug responses [@doi:10.1101/278739].
 The authors trained a variational autoencoder (an unsupervised neural network that learns a series of representations from data), or VAE, on AML data that had been collected over time without the desired phenotypic information (drug response).
-The authors used the learned attributes to encode a low-dimensional representation of held-out AML data with phenotype labels of interest, and used this representation as input to a classifier that predicted _in vitro_ drug response.
-
-
-Though there were over 6500 AML samples from many different studies used as part of the training set in Dincer et al. [@doi:10.1101/278739], we expect that in other rare diseases considerably fewer samples will be available or may be from different tissues in systemic diseases.
-The study by Dincer and colleagues highlights another challenge: samples collected as part of multiple studies may not be associated with the deep phenotypic information that would maximize their scientific value.
-
-
-<!-- From transfer learning -->
-
-Feature-representation-transfer is embodied in Dincer et al., where features are learned from unlabeled AML data and then used to encode a low-dimensional representation of AML data with in vitro drug response labels [@doi:10.1101/278739]. 
-The authors then used this low-dimensional representation as input to predict drug response labels–a supervised example.
+VAEs are distinct from other autoencoders in that the distribution of the encodings are regularized such that they are close to a normal distribution, which can help avoid overfitting and learn a representation that is more than just an identity mapping [TODO: CITATIONS / Take another pass at this sentence].
```

Comment:
It definitely regularizes by adding the additional constraint. However, the extent of that constraint (kinda heavily discussed in the beta-VAE literature since that focuses on weighting the two constraints) and how that produces this outcome is still a bit unclear.  There are a few results in Greg's BioBombe paper ( https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02021-3 ) that also hint at this VAE constraint as being helpful in this way.

Reply:
Yep, this was on my mind! I wasn't sure how to thread this needle in a sentence or two. It may be helpful to just come out and say that some of these things are unclear.